src/main/java/minijava/Cli.java

```diff
@@ -267,10 +277,9 @@ static Parameters parse(String... args) {
       Parameters params = new Parameters();
       try {
         new JCommander(params, args);
-        if (params.mainParameters.size() == 1) {
-          params.file = params.mainParameters.get(0);
-        }
+        params.file = params.mainParameters.get(params.mainParameters.size() - 1);
```

Comment:
This change makes it possible to pass multiple files to the CLI without failing (in validate()), please revert this.